torch/csrc/autograd/profiler_kineto.h

```diff
@@ -2,12 +2,9 @@
 
 #include <torch/csrc/autograd/profiler_legacy.h>
 
-// Kineto is currently available on Linux server-side
-#ifdef USE_KINETO
-#if !defined(__linux__) || defined(_WIN32) || defined(C10_MOBILE) || defined(__APPLE__) || defined(DISABLE_KINETO)
+#ifdef DISABLE_KINETO
```

Comment:
Hmm, do you know the history of this change? I.e. why one would define both `USE_KINETO` and `DISABLE_KINETO`?

Reply:
I think that's a debug remnant, will remove